pkg/sinks/kafka/kafka.go

```diff
@@ -83,14 +81,14 @@ func NewToKafka(vertex *dfv1.Vertex,
 	toKafka.topic = kafkaSink.Topic
 	toKafka.kafkaSink = kafkaSink
 
-	forwardOpts := []forward.Option{forward.WithVertexType(dfv1.VertexTypeSink), forward.WithLogger(toKafka.log)}
+	forwardOpts := []sinkforward.Option{sinkforward.WithLogger(toKafka.log)}
```

Comment:
Do we need to clean up the option `forward.WithVertexType()`?

Reply:
yes but redis read test will fail if we remove it